docs/content/en/docs/getting-started/developer/_index.md

```diff
@@ -0,0 +1,281 @@
+```
+
+In the output above, the service of interest is:
+
+* **k8ssandra-dc1-stargate-service**: The K8ssandra Stargate service where the name is a combination of the K8ssandra cluster name you specified during the Helm install, `k8ssandra`, the datacenter name, `dc1` and the postfix, `-service`. This service listens on the ports:
+  * **8080/TCP**: GraphQL interface
+  * **8081/TCP**: REST authorization service for generating tokens
+  * **8082/TCP**: REST interface
+  * **8084/TCP**: Health check (/healthcheck)
+  * **8085/TCP**: Metrics (/metrics)
+  * **9042/TCP**: CQL service
+
+Those are the ports we'll need to forward for CQLSH and Stargate access.
+
+To configure port forwarding:
+
+1. Open a new terminal.
+
+2. Run the `kubectl port-forward` command in the background:
+
+    ```bash
+    kubectl port-forward svc/k8ssandra-dc1-stargate-service 8080 8081 8082 8084 8085 9042 &
+    ```
+
+    **Output**:
+
+    ```bash
+    [1] 80940
+
+    ~/
+    Forwarding from 127.0.0.1:8080 -> 8080
+    Forwarding from [::1]:8080 -> 8080
+    Forwarding from 127.0.0.1:8081 -> 8081
+    Forwarding from [::1]:8081 -> 8081
+    Forwarding from 127.0.0.1:8082 -> 8082
+    Forwarding from [::1]:8082 -> 8082
+    Forwarding from 127.0.0.1:8084 -> 8084
+    Forwarding from [::1]:8084 -> 8084
+    Forwarding from 127.0.0.1:8085 -> 8085
+    Forwarding from [::1]:8085 -> 8085
+    ```
+
+### Terminate port forwarding
+
+To terminate the port forwarding service:
+
+1. Get the process ID:
+
+    ```bash
+    jobs -l
+    ```
+
+    **Output**:
+
+    ```bash
+    [1]  + 80940 running    kubectl port-forward svc/k8ssandra-dc1-stargate-service 8080 8081 8082 8084
+    ```
+
+1. Kill the process
+
+    ```bash
+    kill 29212
+    ```
+
+    **Output**:
+
+    ```bash
+    [1]  + terminated  kubectl port-forward svc/k8ssandra-dc1-stargate-service 8080 8081 8082 8084
+    ```
+
+{{% alert title="Tip" color="success" %}}
+Exiting the terminal instance will terminate the port forwarding service.
+{{% /alert %}}
+
+## Access K8assandra using the Stargate APIs
+
+[Stargate](https://stargate.io/) provides APIs, data types and access methods that bring new capabilities to existing databases. Currently Stargate adds both REST and GraphQL APIs for CRUD access to data stored in Apache Cassandra® and there are many more APIs coming soon. Separating compute and storage also has benefits for maximizing resource consumption in cloud environments. When using Stargate with Cassandra, you can offload the request coordination overhead from your storage instances onto Stargate instances which has shown latency improvements in preliminary testing.
+
+To access K8ssandra using Stargate:
+
+1. Make sure that all the Stargate services are in order at the Stargate health check endpoint, <http://127.0.0.1:8084/healthcheck>. The pretty-printed raw JSON data should have a value of `true` for each `healthy` key:
```

Comment:
We can avoid the healthcheck endpoint altogether. It's mostly used internally to make sure the pod is up.